<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommentariesWidget</class>
 <widget class="QWidget" name="CommentariesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QListWidget" name="listComments">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>400</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetNewComment" native="true">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="buttonAddComment">
        <property name="text">
         <string>Прокомментировать!</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPlainTextEdit" name="textEditComment"/>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Ваш комментарий:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
